DXRModules/DeusEx/Classes/DXREnemies.uc

```diff
@@ -91,6 +91,7 @@ function CheckConfig()
     AddRandomEnemyType(class'#var(prefix)Gray', 2, MJ12);
     AddRandomEnemyType(class'#var(prefix)SpiderBot2', 2, MJ12);//little spider
     AddRandomEnemyType(class'#var(prefix)SpiderBot', 2, MJ12);//big spider
+    AddRandomEnemyType(class'FrenchGray', 1, MJ12);
```

Comment:
Do we want to do this only for Paris? I think it would be easy and it would make sense (as much sense as possible at least)

Reply:
It could possibly be an alternate instead of the regular only in Paris.  Wouldn't having it be mission-specific mess up the RNG?

Reply:
> It could possibly be an alternate instead of the regular only in Paris. Wouldn't having it be mission-specific mess up the RNG?

Mess up the RNG in what way? Could do a -1 to the regular gray in the same if statement and have an else for the regular gray number